backend/src/controller/team/response/get-teams.ts

```diff
@@ -0,0 +1,35 @@
+import { ApiProperty } from '@nestjs/swagger'
+import { TeamDTO, PairDTO } from 'src/app/query-service-interface/team-qs'
+
+export class GetTeamDataResponse {
+  @ApiProperty({ type: () => [TeamDTO] })
+  teamData: TeamDTO[]
+
+  public constructor(params: { teamData: TeamDTO[] }) {
+    const { teamData } = params
+    this.teamData = teamData.map(({ id, name, pairs }) => {
+      return new Team({
+        id,
+        name,
+        pairs,
+      })
+    })
+  }
+}
+
+class Team {
+  @ApiProperty()
+  id: string
+
+  @ApiProperty()
+  name: string
+
+  @ApiProperty()
```

Comment:
```suggestion
  @ApiProperty({ type: [PairDTO]})
```
こんな感じで書くとswaggerに反映されました

Reply:
配列は明示的に指定する必要があるようです
https://docs.nestjs.com/openapi/types-and-parameters#arrays

Reply:
ありがとうございます！

Reply:
普通に配列を明示してもダメだったので保留にします！
https://github.com/kai815/praha-challenge-extra-large-task/pull/22/commits/504863f9552d9c34358d8fd5bd6940679e9c41c2